[DO NOT MERGE]Update management api documentation (#2834)
wso2/docs-ei: en/micro-integrator/docs/administer-and-observe/working-with-management-api.md

### GET CARBON APPLICATIONS

> Note : "This capability to list down faulty applications is released as a product update on todo. If you don't already have this update, you can [get the latest updates](https://updates.docs.wso2.com/en/latest/updates/overview/#!) now.

Reviewer: ```suggestion
Note: "This capability to list down faulty applications is released as a product update on todo. If you do not have this update, you can [get the latest updates](https://updates.docs.wso2.com/en/latest/updates/overview/#!) now.
```

Reviewer: Let's apply for all similar sentences.

Author: Fixed.

> Note: This capability to list down faulty applications is released as a WUM update on 16/09/2021 and Update 2.0 on 06/09/2021. If you do not have this update, you can use [WSO2 Update Manager(WUM)](https://docs.wso2.com/display/updates/WSO2+Updates) to get the wum update or use [Update 2.0 Tool](https://updates.docs.wso2.com/en/latest/updates/overview/#!) to get the Update 2.0 update now.

Reviewer: Two quick questions:
1. Why is the date different for both updates?
2. Shouldn't these docs be published now since the updates are released now?

Author: 1) The wum update is released on 16/09/2021 and update 2.0 was released 06/09/2021. 
2) We kept this PR on hold until both WUM and u2 update is released.
